scripts/config/webpack/config-client.js

```diff
@@ -231,6 +232,10 @@ module.exports = ({ minify } = {}) => {
                 allChunks: true,
                 disable: isDev,
             }),
+            !isDev && gzip && new CompressionPlugin({
```

Comment:
Related to my previous comment regarding always compressing, I wold say that when minify is active, compression is also active.